gobblin-binary-management/build.gradle

```diff
@@ -25,9 +25,9 @@ dependencies {
   compile externalDependency.avroMapredH2
   compile externalDependency.guava
   compile externalDependency.hadoopHdfs
-  runtime externalDependency.hadoopCommon
-  runtime externalDependency.hadoopClientCore
-  runtime externalDependency.hadoopAuth
+  runtimeOnly externalDependency.hadoopCommon
```

Comment:
Is there a reason all dependencies are mapped into "runtimeOnly"? I suspect that what you really want here is "implementation" (it will be mapped to Maven "runtime" scope in the poms). See https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation

Reply:
so, those dependencies where "runtime" before I changed it to "runtimeOnly" because maven-publish wasn't mapping them into the pom file. RuntimeOnly is an extension of runtime configuration and maven-publish plugin maps it correct "runtime" scope.
Not sure why we need to change it to "implementation"
Reference: 
https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management

Reply:
> so, those dependencies where "runtime" before I changed it to "runtimeOnly" because maven-publish wasn't mapping them into the pom file.

Understood. "implementation" is also mapped to "runtime" scope in poms. Please read the Gradle docs to understand the difference between "implementation" and "runtimeOnly". If you want to continue to use "runtimeOnly" for all those dependencies, I'm OK.

Reply:
Understood! You're right and "runtimeOnly" is the right path forward. Thanks!